internal/twilio/twilio.go

```diff
@@ -0,0 +1,28 @@
+package twilio
+
+import (
+	"fmt"
+	"os"
+	"github.com/twilio/twilio-go"
+	"github.com/joho/godotenv"
+)
+
+var TwilioClient *twilio.RestClient
+var TwilioPhoneNumber string
+
+func init() {
+	err := godotenv.Load(".env")
+
+	if err != nil {
+		fmt.Println("Error loading .env file")
+	}
+
```

Comment:
@JohnUUU please make a note about env vars

Reply:
Note: Env Vars store the Twilio Account SID and password, as well as the phone number that sends the notification. When we actually roll out the notification we probably want to change these to a department approved Twilio account.